Pipfile

```diff
@@ -8,9 +8,11 @@ python_version = "3.9"
 
 [packages]
 dominate = "~=2.6.0"
+click = "8.1.3"
```

Comment:
we have all the other dependencies like this:
```suggestion
click = "~=8.1.3"
```

Reply:
ack